### DISCUSSION TITLE
Privacy sandbox?

### DISCUSSION THREAD

**Discussion by dbui04** [bromite/bromite#1547] [Q&A] [answered]
There's a setting called "privacy sandbox" in the lastest version (95.0.4638.78). As far as I know it's just an equivalent of FLoC, so shouldn't it be removed?

**Comment by uazo** [comment, marked as answer]
> As far as I know it's just an equivalent of FLoC

privacy sandbox is more than FLoC (see https://www.chromium.org/Home/chromium-privacy/privacy-sandbox).

>  so shouldn't it be removed?

FloC and other features are currently disabled. 
Those settings will likely be removed in a future bromite release

**Reply by dbui04** [reply]
Got it. Thanks for your answer.
